cmd/ocm-support/sync-cloud-resources/cmd.go

```diff
@@ -0,0 +1,209 @@
+package sync_cloud_resources
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:          "sync-cloud-resources [branch-name] [csv-path]",
+	Short:        "Syncs cloud resources in AMS and generates quota rules for them",
+	Long:         "Syncs cloud resources in AMS and generates quota rules for them",
+	RunE:         syncCloudResources,
+	SilenceUsage: true,
+	Args:         cobra.ExactArgs(2),
+}
+
+var args struct {
+	dryRun bool
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(&args.dryRun, "dry-run", true, "When true, don't actually take any actions, just print the actions that would be taken")
+}
+
+func syncCloudResources(cmd *cobra.Command, argv []string) error {
+	branchName := argv[0]
+	csvPath := argv[1]
+	if branchName == "" {
+		return fmt.Errorf("branch name cannot be empty")
+	}
+	if csvPath == "" {
+		return fmt.Errorf("csv path cannot be empty")
+	}
```

Comment:
when running without arguments, this block is not being reached:

```
 ./ocm-support sync-cloud-resources
ERRO[0000] accepts 2 arg(s), received 0 
```

Reply:
This is because we expect the branch name and CSV path to be passed
Check ocm-support sync-cloud-resources --help

Reply:
Right, but if we omit the argument in existing commands, the help is shown automatically - which is not happening in this case.